rebuild/lib/index.d.ts

```diff
@@ -88,3 +88,22 @@ export class Signature implements Serializable {
   serialize(compress?: boolean): Buffer;
   sigValidate(): void;
 }
+
+export function aggregatePublicKeys(keys: PublicKeyArg[]): PublicKey;
+export function aggregateSignatures(signatures: SignatureArg[]): Signature;
+export function verify(msg: BlstBuffer, publicKey: PublicKeyArg, signature: SignatureArg): boolean;
+export function asyncVerify(msg: BlstBuffer, publicKey: PublicKeyArg, signature: SignatureArg): Promise<boolean>;
```

Comment:
can we separate out async from the sync and put a new line separator between them

Reply:
Updated.